src/List.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Item from './Item';
+
+export default function List({ tasks, onClickDelete }) {
+  return (
+    <ol>
+      {tasks.map(task => (
+        <Item key={task.id} task={task} onClick={onClickDelete} />
```

Comment:
```suggestion
        <Item 
          key={task.id}
          task={task} 
          onClick={onClickDelete} 
        />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
넵 반영하겠습니다~!